keyboards/cipulot/ec_pro2/ec_pro2.c

```diff
@@ -0,0 +1,159 @@
+/* Copyright 2023 Cipulot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef VIA_ENABLE
+#    include "ec_switch_matrix.h"
+#    include "print.h"
+#    include "eeconfig.h"
+#    include "quantum.h"
```

Comment:
```suggestion
#include "ec_switch_matrix.h"
#include "quantum.h"
```

Reply:
@fauxpark regarding this change, I tested the suggested changes for the `include` and those are working.

About the `#ifdef VIA_ENABLE` and relative `#endif`, I implemented that so that the following functionality is only enabled when VIA is enabled. In case someone compiles the code with `default` as target keymap the default  actuation levels used are assigned in the `matrix.c` file. In here those will be overwritten with the ones in eeprom if VIA enabled.

I'll push for now the changes for the `includes` and wait to apply the removal of the `#ifdef` till your response on this 👍

Reply:
If this code is only used when VIA is enabled, then it should not be unconditionally added to `SRC`.

Reply:
The addition of the `apc.c` file without the condition tho generates errors if I compile the `default` keymap, since it tries to reference variables that are only defined when the VIA stuff is imported.

Reply:
I'll revert the latest change since it breaks the `default` keymap compile.

Reply:
You need to move it to the via keymap's rules.mk. That's what I'm referring to by "unconditionally added to SRC."

Reply:
Ohhhhh, ok, I'm doodo. I'll do it now

Reply:
@fauxpark I've moved the entire functionality inside of the VIA keymap folder, removing the `ec_pro2.c` file and adding the new file containing the functions, `via_apc.c` in the `SRC` of VIA's `rules.mk`. 

Changes tested and working. Now both the via and default keymap targets compile without problems.